com.unity.render-pipelines.high-definition/Runtime/RenderPipelineResources/Mesh/Sphere.fbx.meta

```diff
@@ -1,17 +1,11 @@
 fileFormatVersion: 2
-guid: 9e0af751bc36ea146940ba245193e28c
+guid: f7f013a81640a284ea38df61c509ff9a
```

Comment:
This change of `guid` will lead to a lost references: https://github.com/Unity-Technologies/Graphics/search?q=9e0af751bc36ea146940ba245193e28c 
See this [conversation](https://unity.slack.com/archives/G1BTWN88Z/p1630578902026800).

Reply:
Yup, reverting it as we speak (cannot open the conversation tho :) )